modules/affile/collection-comparator.c

```diff
@@ -125,7 +129,7 @@ collection_comparator_add_initial_value(CollectionComparator *self, const gint64
   entry->value = g_strdup(value);
   entry->flag = IN_OLD;
   self->original_list = g_list_append(self->original_list, entry);
-  g_hash_table_insert(self->original_map, (void*) key, entry);
+  g_hash_table_insert(self->original_map, entry, entry);
```

Comment:
For clarity's sake please use different names for the parameters.

Reply:
the parameters cannot be different, the key and the data is the same